packages/@aws-cdk/aws-sns/lib/topic-base.ts

```diff
@@ -28,7 +28,7 @@ export interface ITopic extends
   /**
    * Subscribe some endpoint to this topic
    */
-  subscribe(name: string, endpoint: string, protocol: SubscriptionProtocol): Subscription;
+  subscribe(name: string, endpoint: string, protocol: SubscriptionProtocol, rawMessageDelivery?: boolean): Subscription;
```

Comment:
Actually this should have been an options object, no?

Reply:
Maybe, yes. I decided against changing this particular signature so drastically at this stage, but feel free to open an issue if you feel strongly about it.